feature-detects/flash.js

```diff
@@ -77,11 +85,18 @@ define(['Modernizr', 'createElement', 'docElement', 'addTest', 'getBody'], funct
           }
         }
 
-        // If we’re rockin’ a fake body, clean it up
-        if (body.fake) {
+        // If we’re rockin’ an attached fake body, clean it up
+        if (body.fake && body.parentNode) {
```

Comment:
Not sure if it stems from this, and I can create an issue for it, but my Modernizr build from gulp-modernizr (after a fresh npm install yesterday) did not remove the extra body element, so we had two `<body>`s in our app!  I had to set modernizr to "3.0.0-pre" in our package.json, since customizr uses the master branch from github.